### PR TITLE
Response body hook

### PR DESCRIPTION
**Pull request by twe4ked**

### CODE REVIEW

File: spec/integration/output/slate/index.html.md [hunk 1]
```diff
@@ -223,3 +223,31 @@ Parameter | Type | Description
 --------- | ------- | -----------
 id | integer | The id of the place
 name | string | The place's name
+
+## Listing all places with a modified response bod,
+
+
+
```
Comment (AeroCross) [inline spec/integration/output/slate/index.html.md]: Extra whitespace in here.

File: spec/integration/output/slate/index.html.md [hunk 2]
```diff
@@ -223,3 +223,31 @@ Parameter | Type | Description
...
+```json
+
```
Comment (AeroCross) [inline spec/integration/output/slate/index.html.md]: And here.

File: spec/integration/output/slate/index.html.md [hunk 3]
```diff
@@ -223,3 +223,31 @@ Parameter | Type | Description
...
+    }
+  ]
+}
+```
+
+### HTTP Request
+
+`GET http://example.com/places`
+
+
```
Comment (AeroCross) [inline spec/integration/output/slate/index.html.md]: And here.

File: spec/integration/rspec_api_docs_spec.rb [hunk 4]
```diff
@@ -171,6 +171,20 @@ def app
...
+        it 'returns all places but only shows one' do
+          doc do
+            name 'Listing all places with a modified response bod,'
+            response_body_after_hook -> (parsed_response_body) {
+              parsed_response_body[:data].delete_if { |i| i[:id] == 1 }
+              parsed_response_body
+            }
+          end
+
+          get '/places'
```
Comment (AeroCross) [inline spec/integration/rspec_api_docs_spec.rb]: I see no expectation here?

File: spec/integration/output/slate/index.html.md (hunk 1 above)
Comment (twe4ked) [inline spec/integration/output/slate/index.html.md]: All the whitespace comments are for the generated files that I don't control the format of. This one is for use with [slate](https://github.com/lord/slate).

File: spec/integration/rspec_api_docs_spec.rb (hunk 4 above)
Comment (twe4ked) [inline spec/integration/rspec_api_docs_spec.rb]: These are used to generate the `spec/integration/output/` files.

**Pull request merged**